x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/action/EqlSearchRequest.java

```diff
@@ -297,6 +316,21 @@ public EqlSearchRequest indices(String... indices) {
         return this;
     }
 
+    @Override
+    public boolean allowsCrossProject() {
+        return true;
+    }
+
+    @Override
+    public void setResolvedIndexExpressions(ResolvedIndexExpressions expressions) {
```

Comment:
It's still not clear if we really need resolved expressions, probably we can avoid this.